src/python/pants/backend/python/rules/pytest_runner.py

```diff
@@ -218,7 +219,7 @@ async def setup_pytest_for_target(
     )
 
 
-@rule
+@rule(level=LogLevel.INFO)
```

Comment:
Rather than adjusting the level of this `@rule`, you probably want to adjust `coordinator_of_tests`. But it should probably stay debug by default: we don't want it to render unless this fails. Finally, rather than changing the name, you should probably give it a description (`desc = 'Run test target'`).

I recognize that we still need to figure out what to do about `Process`es being at INFO by default vs rules being able to affect workunits like this... but in the meantime, there isn't a reason to render a `@rule` unless it fails.

Reply:
I deliberately changed the level to INFO so that it would conitnue to print out a visible success message - but on second though, yeah, we don't need that for successful test runs.

Reply:
We already have that by virtue of the process completing at INFO. So in the case where a test fails, the process will log at INFO, and then the `TestResult` will cause some other workunits to render errors.